Everest/Everest/Models/User.swift

```diff
@@ -45,9 +45,9 @@ class User: NSObject {
     if let momentIds = user["momentIds"].dictionary {
       self.momentIds = momentIds.map { $1.string! }
     }
-    if let ideas = user["ideas"].array {
-      self.ideas = ideas.map { Act(act: $0) }
-    }
+//    if let ideas = user["ideas"].array {
+//      self.ideas = ideas.map { Act(id: String, title: <#String#>) }
+//    }
```

Comment:
do we need this code? whats an idea?

Reply:
This code is to remind us to add a functionality for user to suggest new acts.

Reply:
ok then we should refer to it as a userAct so its relatable.

Reply:
sure